examples/minimal-sveltekit/src/lib/utils/uploadthing.ts

```diff
@@ -0,0 +1,6 @@
+import { generateSvelteHelpers } from "@uploadthing/svelte";
```

Comment:
Nit: it's coming from the svelte package, calling this `generateSvelteHelpers` is a bit redundant. 

We do the same thing in the React package though I guess 🤔

Reply:
leave as is for now. may rethink this in the future, but it should not be locking for this PR